extensions/amp-web-push/validator-amp-web-push.protoascii

```diff
@@ -57,6 +57,13 @@ tags: {  # <amp-web-push>
       allow_relative: false
     }
   }
+  attrs: {
+    name: "service-worker-scope"
+    value_url: {
+      protocol: "https"
```

Comment:
Can a scope be an absolute URL?

Reply:
The [MDN Docs](https://developer.mozilla.org/en-US/docs/Web/API/ServiceWorkerContainer/register#Parameters) say: "This is **usually** a relative URL", but doesn't say **it has to be** a relative URL. 
That's why I assume the answer is yes.

I also wrote [this test](https://amp-web-push-changes.glitch.me/subpath/index.html), which uses an absolute URL for the "service-worker-scope" attribute, and it looks like it's working well when inspecting the installed SW in CDT "Application" tab.

If you want to double-check, here's [the code of the Glitch test](https://glitch.com/~amp-web-push-changes).